input.scss

```diff
@@ -33,9 +33,8 @@ $vui-input-padding: px-to-base-rem(8) !default;
 			padding: calc(0.4rem - 1px);
 		}
 
-		&:disabled {
-			background-color: $vui-color-regolith;
-			border-color: $vui-color-titanius;
+		&:disabled,
```

Comment:
Kind of, it's just the [disabled pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/%3Adisabled)... so the CSS would be: `select.vui-input:disabled`